ui/src/experiments/details/ExperimentDetailsView.js

```diff
@@ -105,18 +119,19 @@ const ExperimentDetailsView = ({ projectId, experimentId, ...props }) => {
                 </ExperimentActions>
               </EuiPageTemplate.Header>
             </Fragment>
-          )}
+          )
+          }
```

Comment:
Formatting looks funny here 🤔

Reply:
Something was broken with my local hooks, fixed it now.